go.mod

```diff
@@ -71,4 +71,8 @@ replace (
 
 	// pinned because no tag supports 1.18 yet
 	sigs.k8s.io/structured-merge-diff => sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06
+
+	// pinned for webhook authorizer panic fix
+	// https://github.com/kubernetes/kubernetes/pull/97820
+	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20210108210607-7e2afe673ceb
```

Comment:
Have you tried using `go get k8s.io/apiserver@7e2afe673ceb` instead of adding a `replace` directive? I would rather us bump dependencies normally than use `replace` where possible.

Reply:
> 
> 
> The 1.20 backport merged ([kubernetes/kubernetes@13295e5](https://github.com/kubernetes/kubernetes/commit/13295e5eddfb7fa2aea4b6f048eb86adc9af92b4)). Can we pin to that commit instead to avoid inadvertently pulling in any 1.21 changes?

Pinned 25d1e3c959d instead, that is on the release-1.20 branch, thanks for pointing that out